mw/xml_dump/iteration/iterator.py

```diff
@@ -140,6 +145,9 @@ def load_site_info(cls, element):
         namespaces = {}
 
         for sub_element in element:
+
+            if sub_element.tag == 'siteinfo':
+                return(cls.load_site_info(sub_element))
```

Comment:
Do you mean is language data inside a tag? 
Unfortunately not. The language is in the xml header, not a tag.